homeassistant/components/squeezebox/media_player.py

```diff
@@ -234,11 +212,24 @@ def __init__(self, player: Player, server: Server) -> None:
             identifiers={(DOMAIN, self._attr_unique_id)},
             name=player.name,
             connections={(CONNECTION_NETWORK_MAC, self._attr_unique_id)},
-            via_device=(DOMAIN, server.uuid),
```

Comment:
Why?

Reply:
Good catch, there is no reason to remove that. I just need to store it in the player coordinator object so that we have access to it now that the server-interfacing code has all been moved to coordinator.py. Which makes sense because we'll want it for other sensors we connect through that server too.